packages/nimble-components/src/theme-provider/tests/tokens.stories.ts

```diff
@@ -185,3 +186,14 @@ export const themeAwareTokens: StoryObj<TokenArgs> = {
 };
 
 themeAwareTokens.storyName = 'Theme-aware Tokens';
+
+// Setting token default values is done as part of the FAST render queue so it needs to be cleared before reading them
+// https://github.com/microsoft/fast/blob/bbf4e532cf9263727ef1bd8afbc30d79d1104c03/packages/web-components/fast-foundation/src/design-token/custom-property-manager.ts#LL154C3-L154C3
+// This uses Storybook's "loaders" feature to await the queue. https://storybook.js.org/docs/html/writing-stories/loaders
+themeAwareTokens.loaders = [
```

Comment:
Also I didn't notice this before but there is a known limitation listed on the [loaders page](https://storybook.js.org/docs/html/writing-stories/loaders#known-limitations) that says loaders are not compatible with the inline-rendered storybook docs: https://github.com/storybookjs/storybook/issues/12726

So may need to double check that this change also helps the docs tab in storybook if this issue manifests there as well.

Reply:
I tried it out in the local build and [pipeline storybook build](https://60e89457a987cf003efc0a5b-podsbmrkls.chromatic.com/?path=/docs/tokens-theme-aware-tokens--theme-aware-tokens) and it looks like it mysteriously works 👻